controllers/topology/OWNERS

```diff
@@ -0,0 +1,8 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - cluster-api-topology-maintainers
```

Comment:
This should also include the normal cluster api maintainers and approvers list

Reply:
First option would require dropping `cluster-api-reviewers` from low-level OWNER files. I can do that. Your answer seems to apply that I should do that? :) (Just a big confused because of the approve/lgtm, or in a follow-up PR?)

Reply:
Just keep it as-is seems fine to me